<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>17</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Aplikasi Kampus</string>
  </property>
  <widget class="QFrame" name="frameMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frameMain { background-color: rgb(0, 0, 0); }</string>
   </property>
   <widget class="QLabel" name="labelHeader">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background-color: rgb(85, 0, 127); border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>ğŸ“Š Aplikasi Kampus</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frameSidebar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>578</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QFrame { background-color: rgb(85, 0, 127); border-radius: 10px; }
      QPushButton { color: white; background-color: #2C3E50; border: none; text-align: left; padding: 10px 20px; border-radius: 8px; font: 10pt 'Segoe UI'; }
      QPushButton:hover { background-color: rgb(85, 170, 255); }
     </string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelTitle_2">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>Aplikasi Kampus</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDashboard">
       <property name="text">
        <string>ğŸ“Š Dashboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUsers">
       <property name="text">
        <string>ğŸ‘¥ Users</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMahasiswa">
       <property name="text">
        <string>ğŸ“ Mahasiswa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnJurusan">
       <property name="text">
        <string>ğŸ“’ Jurusan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKrs">
       <property name="text">
        <string>ğŸ“ KRS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNilai">
       <property name="text">
        <string>ğŸ’¯ Nilai</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMataKuliah">
       <property name="text">
        <string>ğŸ“š Mata Kuliah</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSettings">
       <property name="text">
        <string>âš™ Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAbout">
       <property name="text">
        <string>â„¹ About</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>521</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border-radius: 10px;</string>
    </property>
    <widget class="QWidget" name="pageDashboard">
     <widget class="QLabel" name="lblWelcome">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>521</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Selamat Datang di Sistem Kampus</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
